code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -317,6 +319,8 @@
 	desc = "One of countless obsolete ballistic rifles that still sees use as a cheap deterrent. Uses 10mm ammo and its bulky frame prevents one-hand firing."
 	icon_state = "surplus"
 	inhand_icon_state = "moistnugget"
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'
```

Comment:
this is already handled by build_worn_icon

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.